### PR TITLE
Notify about corrupted files

### LINKED ISSUE #124: Prompt To Clear Corrupted Tags
See #116:
> maybe we could have Tagger ask the user to clear the tags of corrupted files if any, that way you can re-tag them and use them again...

### PR DESCRIPTION
**Pull request by nlogozzo**
Closes #124

### CODE REVIEW

File: NickvisionTagger.GNOME/Views/MainWindow.cs [hunk 1]
```diff
@@ -1233,4 +1236,23 @@ private void CopyFingerprintToClipboard(Gtk.Button sender, EventArgs e)
         _fingerprintLabel.GetClipboard().SetText(_fingerprintLabel.GetText());
         _toastOverlay.AddToast(Adw.Toast.New(_("Fingerprint was copied to clipboard.")));
     }
+
+    /// <summary>
+    /// Occurs when there are corrupted music files found in a music folder
+    /// </summary>
+    private bool CorruptedFilesFound()
+    {
+        var dialog = new MessageDialog(this, _controller.AppInfo.ID, _("Corrupted Files Found"), _("This music folder contains music files whose tags are corrupted. What would you like to do?"), _("Ignore Files"), _("Clear Tags"));
```
Comment (fsobolev) [inline NickvisionTagger.GNOME/Views/MainWindow.cs:1245]: How about "...What would you like to do **with corrupted files**?" so it's understandable that only corrupted tags will be cleared/ignored.

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: NickvisionTagger.GNOME/Views/MainWindow.cs (hunk 1 above)
Comment (nlogozzo) [reply NickvisionTagger.GNOME/Views/MainWindow.cs:1245]: Agreed :)

Could you make the change and update the po files?

Comment (fsobolev) [reply NickvisionTagger.GNOME/Views/MainWindow.cs:1245]: okay, sure :)

**Review by fsobolev — APPROVED** [review]

**Comment by nlogozzo** [comment]
Could you merge main into this branch, remove the ClearCorruptedFilesAsync function from MusicFolder, then run GeneratePot and UpdatePo, and then I'll merge :)

**Comment by fsobolev** [comment]
@nlogozzo Done

**Pull request merged**